Deprecated Security and Compliance – Content Search V2 integration (#44387)
demisto/content: Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

# Microsoft Security & Compliance Center - Content Search Integration

## ⚠️ Deprecation Notice

> This integration is **deprecated**. Customers should transition to the **Microsoft Graph Security** integration for equivalent functionality.

Reviewer: ```suggestion
This integration is deprecated. Use the Microsoft Graph Security integration for equivalent functionality.
```
